### DISCUSSION TITLE
Tailwind prefix support

### DISCUSSION THREAD

**Discussion by danielwyb** [ben-rogerson/twin.macro#345] [General] [answered]
Great plugin, we'd love to use it but our tailwind config includes a prefix (e.g. `tw-<classname>`), which does not seem to be respected by the plugin and causes a compilation error.

Is this something you have on the radar, or could you point me in the right direction to contribute to make this possible?

Thanks!

**Comment by ben-rogerson** [comment]
Okay, let me see what I can do - perhaps I can work something into the next release 👍

**Reply by danielwyb** [reply]
Thanks for adding it, really appreciated, tried it works great :)

**Comment by ben-rogerson** [comment, marked as answer]
Prefix support was added in [twin.macro@2.3.0](https://github.com/ben-rogerson/twin.macro/releases/tag/2.3.0)
